### ISSUE TITLE
AttributeError: 'NoneType' object has no attribute 'display_name'

### ISSUE THREAD

**Issue by e-quate** [ladybug-tools/honeybee-doe2#83]
Tested a previous version ( 0.11.7) and did not have an issue. The new update produces this error:

```
File "C:\Users\smare\AppData\Local\Programs\Python\Python311\Lib\site-packages\honeybee_doe2\writer.py", line 214, in honeybee_model_to_inp
    inp_model = model_to_inp(model, hvac_mapping=hvac_mapping,exclude_interior_walls=exclude_interior_walls)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
File "C:\Users\smare\AppData\Local\Programs\Python\Python311\Lib\site-packages\honeybee_doe2\writer.py", line 154, in model_to_inp
    hb_model.properties.doe2.week_scheduels,
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
File "C:\Users\smare\AppData\Local\Programs\Python\Python311\Lib\site-packages\honeybee_doe2\properties\model.py", line 126, in week_scheduels
    return self._get_week_scheduels(self.host)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
File "C:\Users\smare\AppData\Local\Programs\Python\Python311\Lib\site-packages\honeybee_doe2\properties\model.py", line 136, in _get_week_scheduels
    WeekScheduleDoe.from_schedule_ruleset(
File "C:\Users\smare\AppData\Local\Programs\Python\Python311\Lib\site-packages\honeybee_doe2\properties\activitydescription.py", line 189, in from_schedule_ruleset
    days.append(short_name(myruleset.winter_designday_schedule.display_name)) if myruleset.winter_designday_schedule.display_name \
                                                                                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ 
```

**Comment by mostaphaRoudsari** [comment]
Hi @e-quate, thank you for reporting the issue. Can you share a sample file with us to recreate it. Thanks!

**Comment by e-quate** [comment]
[hbjson.zip](https://github.com/ladybug-tools/honeybee-doe2/files/14297868/hbjson.zip)

**Comment by mostaphaRoudsari** [comment]
The issue is not fully resolved! I'm re-opening this.

**Comment by mostaphaRoudsari** [comment]
The translation should go through now. The schedules are not translated correctly yet, but that's a different problem.

Resolved via #85
